### PR TITLE
Compress layout of plots tab

### PR DESCRIPTION
**Pull request by alisman**

### CODE REVIEW

File: src/routes.jsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ export const makeRoutes = (routing) => {
                 <Route path="/loading/comparison" onEnter={()=>{$(document).scrollTop(0)}} component={GroupComparisonLoading} />
                 <Route path="/results/legacy_submission" onEnter={handleLegacySubmission} component={getBlankPage()} />
 
-                <Route path="/results(/:tab)" onEnter={()=>{$(document).scrollTop(0)}} getComponent={lazyLoadComponent(ResultsViewPage)} />
+                <Route path="/results(/:tab)" onEnter={()=>{}} getComponent={lazyLoadComponent(ResultsViewPage)} />
```
Comment (adamabeshouse) [inline src/routes.jsx]: why?

File: vendor-bundles.webpack.config.js [hunk 2]
```diff
@@ -28,13 +28,11 @@ let config = {
...
     },
 
-    devtool : 'source-map',
-
     plugins: [
 
     ],
 
-    devtool : 'source-map'
+    devtool : ''
```
Comment (adamabeshouse) [inline vendor-bundles.webpack.config.js]: why?

Comment (alisman) [reply vendor-bundles.webpack.config.js]: attempt to reduce built time

Comment (adamabeshouse) [reply vendor-bundles.webpack.config.js]: wont it make it harder to debug?

File: src/routes.jsx (hunk 1 above)
Comment (alisman) [reply src/routes.jsx]: because when user switches tabs on results view, Niki wants browser to stay where it is, not go to top.

Comment (adamabeshouse) [reply src/routes.jsx]: ah okay

**Pull request merged**